manageiq-operator/cmd/manager/main.go

```diff
@@ -84,7 +84,11 @@ func main() {
 
 	ctx := context.TODO()
 	// Become the leader before proceeding
-	err = leader.Become(ctx, "manageiq-operator-lock")
+	operatorName := os.Getenv("OPERATOR_NAME")
+	if operatorName == "" {
+		operatorName = "manageiq-operator"
+	}
+	err = leader.Become(ctx, operatorName+"-lock")
```

Comment:
```suggestion
	err = leader.Become(ctx, operatorName + "-lock")
```

Reply:
I had that, but this is what gofmt wants. 🤷‍♂️

Reply:
```suggestion
        lockName := operatorName + "-lock"
	err = leader.Become(ctx, lockName)
```

Reply:
yep, had that too, nerve of me wanting to save a line 😉 I'll push an update.